superverse/__init__.py

```
@@ -6,115 +6,176 @@
+    "plot_images_grid",
+    "polygon_to_mask",
+    "polygon_to_xyxy",
+    "process_video",
+    "resize_image",
+    "rle_to_mask",
+    "scale_boxes",
+    "scale_image",
+    "xcycwh_to_xyxy",
+    "xywh_to_xyxy",
+    "xyxy_to_polygons",
+    "xyxy_to_xyah",
```

Pylint (3.3.7)

[error] Undefined variable name 'xyxy_to_xyah' in __all__

(E0603)